CONTRIBUTING.md

```diff
@@ -46,6 +47,10 @@ CNCF has many existing projects and new ones are submitted to [sandbox](https://
 
 Put yourself on the agenda for an upcoming meeting to present one of these project reviews.
 
+## Present your project for sandbox review
+
+If you want to present your project in one of the next TAG meetings, please create an issue wishing to add your project to the TAG meeting agenda in this repo and tag it with `project` and `tag-discuss` linking your [sandbox issue](https://github.com/cncf/sandbox/issues in the issue description.
```

Comment:
Thanks for this @Jenniferstrej! Should we perhaps suggest that any open project related to cloud-native app delivery is good to present? It could be "Present your project" even if not for sandbox review. We discussed at Kubecon how sometimes folks have good ideas but may not be up to building them into a full, maintained project.

Still would be good to emphasize submitted projects somehow though! Maybe we could say something like, "If you've submitted this project to CNCF please let us know and refer to your [sandbox issue](https://github.com/cncf/sandbox/issues) too!"

Reply:
BTW, this is a question and an important (mandatory I guess based on TOC feedback) point in the Sandbox submission form, see:
https://github.com/cncf/sandbox/issues/new?assignees=&labels=New&projects=&template=application.yml&title=%5BSandbox%5D+%3CProject+Name%3E
```
Project presentations
Has your project been presented to any TAG? If so, please link meeting notes and/or recordings as applicable.
```

Reply:
BTW, this is a question and an important (mandatory I guess based on TOC feedback) point in the Sandbox submission form, see:
https://github.com/cncf/sandbox/issues/new?assignees=&labels=New&projects=&template=application.yml&title=%5BSandbox%5D+%3CProject+Name%3E
```
Project presentations
Has your project been presented to any TAG? If so, please link meeting notes and/or recordings as applicable.
```